js/script.js

```diff
@@ -1,290 +1,127 @@
+        }
+        member.detail = memberDetail;
+        if (committee.members.length) {
+          var html = templates.member({ cmte: committee });
+          $('#committee-list').html(html);
+          currentCommittee = committee;
         }
       })
-        .done(function(memberDetail) {
-          if (committee.members[i].role) {
-            if (committee.members[i].role.toLowerCase() === "member") {
-              committee.members[i].role = null;
-            } else {
-              committee.members[i].role = committee.members[i].role.capitalize();
-            }
-          }
-          // just first letter of party
-          if (memberDetail.party) {
-            memberDetail.party = memberDetail.party.slice(0, 1);
-          }
-          member.detail = memberDetail;
-          if (committee.members.length) {
-//                        $('#update1-left pre').append(JSON.stringify(committee, null, 2));
-            listMembers(committee);
-          }
-        })
     }
-  })
+  });
 };
 
-function listMembers(committee) {
-  var context = {
-    cmte: committee
-  };
-  var html = app.memberTemplate(context);
-  $('#committee-list')
-    .html(html);
-  var memberDetail;
-  //  $("[data-member-id]").on("click", function(e) {
-  jQuery(document.body)
-    .on("click", "[data-member-id]", function(e) {
-      var ID = $(this)
-        .data("member-id");
-      memberDetail = _.findWhere(committee.members, {
-        leg_id: ID
-      });
-      memberDetailFunction(memberDetail);
-    });
-}
-
-function memberDetailFunction(mDetail) {
-  var memberContext = {
-    members: mDetail instanceof Array ? mDetail : [mDetail]
-  };
-  var htmlDetail = app.memberDetail(memberContext);
-  if (memberContext.members.length > 0) {
-    $('#member-detail')
-      .html(htmlDetail);
-  } else {
-    $('#member-detail')
-      .html('<p>Click a committee member<br />for detail. <br />Click the <i class="fa fa-link"></i> icon to visit the committee website. (It will open in a new window.) </p>');
-  }
-}
-$(document.body)
-  .on("click", "[data-cmte-id]", function(e) {
-    e.preventDefault();
-    var committee_id = $(this)
-      .data("cmte-id");
-    queryString.push('committee', committee_id);
-
-    console.log('data-committee', committee_id, 'queryString', queryString);
-    $(this)
-      .parent()
-      .parent()
-      .css('left', '-99999px')
-      .removeClass("open");
-    $(".panel")
-      .html("Click a committee member for detail. <i class='fa fa-long-arrow-right'></i>");
-    getCommitteeDetail(committee_id);
-  });
-// http://css-tricks.com/snippets/javascript/get-url-variables/
-function getQueryVariable(variable) {
-  var query = window.location.search.substring(1);
-  var vars = query.split("&");
-  for (var i = 0; i < vars.length; i++) {
-    var pair = vars[i].split("=");
-    if (pair[0] == variable) {
-      return pair[1];
+$(document.body).on("click", "[data-member-id]", function() {
```

Comment:
I moved this out of `listMembers`, because otherwise you're adding a new listener every time you list members, and that's going to cause repeated renderings until the page bogs down.